src/_data/plugins/eleventy-plugin-postcss.json

```diff
@@ -0,0 +1,6 @@
+{
+    "npm": "eleventy-plugin-postcss",
+    "author": "pewstiepoll",
```

Comment:
@pewstiepoll Is this a Twitter handle? I'm getting an error when I try and check https://twitter.com/pewstiepoll

Reply:
@pdehaan Oops, I thought it was a Github URL. Removed in the new commit.